### PR TITLE
Add more assets from exchanges

### PR DESCRIPTION
**Pull request by yabirgb**

### CODE REVIEW

File: updates/3/updates.sql [hunk 1]
```diff
@@ -49,4 +49,38 @@ INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingec
...
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xaA8330FB2B4D5D07ABFE7A72262752a8505C6B37", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xaA8330FB2B4D5D07ABFE7A72262752a8505C6B37", "C", "Polka City", "POLC", 1613575886, NULL, "polkacity", "", "0xaA8330FB2B4D5D07ABFE7A72262752a8505C6B37");
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("WIN-4", "P", "WINkLink", "WIN", 1564440570, NULL, "winklink", "", "WIN-4"); INSERT INTO common_asset_details(asset_id, forked) VALUES("WIN-4", NULL);
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("STX", "W", "Stacks", "STX", 1572055200, NULL, "blockstack", "STX", "STX"); INSERT INTO common_asset_details(asset_id, forked) VALUES("STX", NULL);
```
Comment (LefterisJP) [inline updates/3/updates.sql:83]: This is already in the global DB @yabirgb, with the identifier STX-2. Please before adding an asset make sure to check that we do not have it added already.

File: updates/3/updates.sql [hunk 2]
```diff
@@ -49,4 +49,38 @@ INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingec
...
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("NDAU", "W", "Ndau", "NDAU", 1569891600, NULL, "ndau", "XND", "NDAU"); INSERT INTO common_asset_details(asset_id, forked) VALUES("NDAU", NULL);
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xaA8330FB2B4D5D07ABFE7A72262752a8505C6B37", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xaA8330FB2B4D5D07ABFE7A72262752a8505C6B37", "C", "Polka City", "POLC", 1613575886, NULL, "polkacity", "", "0xaA8330FB2B4D5D07ABFE7A72262752a8505C6B37");
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("WIN-4", "P", "WINkLink", "WIN", 1564440570, NULL, "winklink", "", "WIN-4"); INSERT INTO common_asset_details(asset_id, forked) VALUES("WIN-4", NULL);
```
Comment (LefterisJP) [inline updates/3/updates.sql:81]: I think this is also already in the DB as `WIN-3`:

```
sqlite> SELECT * from assets WHERE symbol="WIN";
_ceth_0xBfaA8cF522136C6FAfC1D53Fe4b85b4603c765b8|C|WinToken|WIN|1528126931|||WINT|0xBfaA8cF522136C6FAfC1D53Fe4b85b4603c765b8
_ceth_0x899338b84D25aC505a332aDCE7402d697D947494|C|WCOIN|WIN|1514073600||wawllet|WCOIN|0x899338b84D25aC505a332aDCE7402d697D947494
WIN-3|P|WINk|WIN|1562532570||wink|WIN|WIN-3
```

File: updates/3/updates.sql [hunk 3]
```diff
@@ -49,4 +49,38 @@ INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingec
...
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090", "C", "CPChain", "CPC", 1516127905, NULL, "cpchain", "CPC", "0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xc528c28FEC0A90C083328BC45f587eE215760A0F", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xc528c28FEC0A90C083328BC45f587eE215760A0F", "C", "Endor Protocol Token", "EDR", 1527487398, NULL, "endor", "EPT", "0xc528c28FEC0A90C083328BC45f587eE215760A0F");
```
Comment (LefterisJP) [inline updates/3/updates.sql:61]: This is already there. With exact same adddress even. That means the update will fail. Did you try to do the update before opening the PR?

```
sqlite> SELECT * from assets WHERE symbol="EDR";
_ceth_0xc528c28FEC0A90C083328BC45f587eE215760A0F|C|Endor Protocol|EDR|1527465600||endor|EPT|0xc528c28FEC0A90C083328BC45f587eE215760A0F
EDR-2|B|E-Dinar Coin|EDR|1472226274||||EDR-2
```

File: updates/3/updates.sql [hunk 4]
```diff
@@ -49,4 +49,38 @@ INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingec
...
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x6F919D67967a97EA36195A2346d9244E60FE0dDB", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x6F919D67967a97EA36195A2346d9244E60FE0dDB", "C", "Blockcloud", "BLOC", 1554704948, NULL, "blockcloud", "BLOC", "0x6F919D67967a97EA36195A2346d9244E60FE0dDB");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090", "C", "CPChain", "CPC", 1516127905, NULL, "cpchain", "CPC", "0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090");
```
Comment (LefterisJP) [inline updates/3/updates.sql:59]: This is already there too. Exact same address.

```
sqlite> SELECT * from assets WHERE symbol="CPC";
_ceth_0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090|C|CPChain|CPC|1516060800||cpchain|CPCH|0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090
CPC-2|B|Capricoin|CPC|1437350400||capricoin|CPC|CPC-2
```

Update won't work.

File: updates/3/updates.sql [hunk 5]
```diff
@@ -49,4 +49,38 @@ INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingec
...
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xB8E2e2101eD11e9138803cd3e06e16dd19910647", 2, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xB8E2e2101eD11e9138803cd3e06e16dd19910647", "C", "ArdCoin", "ARDX", 1566984871, NULL, "ardcoin", "ARDX", "0xB8E2e2101eD11e9138803cd3e06e16dd19910647");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x6F919D67967a97EA36195A2346d9244E60FE0dDB", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x6F919D67967a97EA36195A2346d9244E60FE0dDB", "C", "Blockcloud", "BLOC", 1554704948, NULL, "blockcloud", "BLOC", "0x6F919D67967a97EA36195A2346d9244E60FE0dDB");
```
Comment (LefterisJP) [inline updates/3/updates.sql:57]: This is already there

```
sqlite> SELECT * from assets WHERE symbol="BLOC";
_ceth_0x6F919D67967a97EA36195A2346d9244E60FE0dDB|C|Blockcloud|BLOC|1540598400||blockcloud||0x6F919D67967a97EA36195A2346d9244E60FE0dDB
```

**Review by LefterisJP — CHANGES_REQUESTED** [review]
You tried to add quite a few assets that already exist in the DB. Please do check that the asset you are adding does not already exists in the DB first ...

What you need to do to fix the "not found" messages in the exchange when the asset already exists is to **explicitly** specify the mapping. Rotki has no way of knowing what `WIN` means for an exchange.

File: updates/3/updates.sql (hunk 1 above)
Comment (yabirgb) [reply updates/3/updates.sql:83]: I only checked by name and not by symbol. I have `STX-2` as Blockstack and not Stacks

File: updates/3/updates.sql (hunk 2 above)
Comment (yabirgb) [reply updates/3/updates.sql:81]: I saw a different coingecko id and assumed they were different. Seems like win redirects winlink

File: updates/3/updates.sql (hunk 1 above)
Comment (LefterisJP) [reply updates/3/updates.sql:83]: Check with direct SQL queries. The search in the app is by text and afaik, @kelsos wants to fix this

File: updates/3/updates.sql (hunk 4 above)
Comment (yabirgb) [reply updates/3/updates.sql:59]: The cryptocompare is wrong. `CPCH` -> `CPC` needs to be an update

**Review by LefterisJP — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**